cirq-core/cirq/circuits/circuit.py

```diff
@@ -1769,6 +1773,12 @@ def __init__(
             else:
                 self.append(flattened_contents, strategy=strategy)
 
+    def _mutated(self) -> None:
```

Comment:
shouldn't this also include `_is_measurement` and `_parameter_names`?

Reply:
Good catch. Fixed.